projects/blenderbot2/agents/blenderbot2.py

```diff
@@ -290,7 +299,7 @@ def add_cmdline_args(
         )
         bb2_group.add_argument(
             '--memory-extractor-phrase',
-            type=str,
+            type='nonestr',
```

Comment:
I haven't seen this before. Does it mean none or str?

Reply:
Yes it means none or str: https://github.com/facebookresearch/ParlAI/blob/304fe1c61fc135a76c03555b45f04df0f4550805/parlai/core/torch_agent.py#L705